egs/avspeech/README.md

```diff
@@ -0,0 +1,14 @@
+### Installed 
+- youtube-dl (pip install youtube-dl)
+- cv2 (pip install opencv-python)
+- facenet_pytorch ()
+- pysndfx
+- catalyst
+- mir_eval
+- alchemy: pip install alchemy-catalyst
+
+### Notes: 
+- Download Interruptible but starts from scratch if interrupted.
+- Need storage_dir variable even if not env variable : 
+    set it where needed.
+    
```

Comment:
```suggestion
```

Reply:
Removed my notes. 
We should add a small description of the dataset here and link to the paper.